packages/pn-personagiuridica-webapp/src/pages/NuovaDelega.page.tsx

```diff
@@ -254,7 +254,7 @@ const NuovaDelega = () => {
                                 id="select-pf"
                                 onClick={() => deleteInput(setFieldValue, setFieldTouched)}
                                 value={RecipientType.PF}
-                                control={<Radio />}
+                                control={<Radio id="select-pf" />}
```

Comment:
Same as before. The id is set on parent component. Why do we have to set it also on child component?

Reply:
i insert id in child of. radio because the id 'select-pf' place in div which contain the child and not child

Reply:
same as before